docs/source/en/package_reference/inference_endpoints.md

```diff
@@ -0,0 +1,48 @@
+# Inference Endpoints
+
+Inference Endpoints offers a secure production solution to easily deploy any `transformers`, `sentence-transformers`, and `diffusers` models from the Hub on a dedicated and autoscaling infrastructure managed by Hugging Face.
+
+An Inference Endpoint is built from a model from the [Hub](https://huggingface.co/models). When an Endpoint is created, the service creates image artifacts that are either built from the model you selected or a custom-provided container image. The image artifacts are completely decoupled from the Hub source repositories to ensure the highest security and reliability levels.
```

Comment:
Docker images are not created on deploy, we load the model in a init container from a base image we built or the image provided by the user.